pages/knowledge.md

```diff
@@ -80,13 +80,29 @@ treasure room, etc.
 ## Cairns of Passage and Return
 
 <div class="surfacePane" markdown="1">
-Note: these are called "Cairns" in PotD, "Beacons"  in HoH, and "Pylons" in EO.
+Note: these are called "Cairns" in PotD, "Beacons" in HoH, and "Pylons" in EO.
 
 The number of enemies you must kill to activate the Cairns is random, with a
 range that varies depending on the floorset. The range for the Cairn of Passage
 is listed on each floorset page. The Cairn of Return generally has a lower
 limit of 2-3 kills more than the Cairn of Passage. Both Cairns likely have the
-same upper limit for a given floorset, but this is unverified.
+same upper limit for a given floorset, but this is unverified. The Pomander of
+Flight causes each kill to count double toward activation.
+
+The random kill counts seem to follow a roughly normal distribution, such that
+numbers in the middle of the range are more common than the low or high ends.
+PotD/HoH are very slightly biased toward lower kill counts.
+
+Killing all enemies on a floor will instantly unlock both Cairns, regardless
+of the number of kills originally required. If you kill all enemies and the
+Cairns don't unlock, then either a new enemy spawned before you got the last
+kill, or an enemy is stuck inside a wall somewhere.
```

Comment:
I'm not sure this is true. I've definitely had to wait for multiple respawns before, especially on low level PotD floors.

Reply:
And since it's not fair to ask you for evidence on kills/floor and not provide any here - log from PotD floor 11 (thus no Flight) opening the Cairn of Return (normally requires at least 5 kills) with a full floor clear of 4 kills:
<img width="413" height="305" alt="image" src="https://github.com/user-attachments/assets/0fd43be4-9866-4955-b70a-a25554d3d2c9" />